### DISCUSSION TITLE
GPT-SoVITSV2什么时候能用呢？是否可以支持--compile

### DISCUSSION THREAD

**Discussion by tuxiaoseng** [lenML/Speech-AI-Forge#197] [想法 | Ideas]
想用来做服务，gpt速度情感还是不错的，别的有点太慢了，是否可以把gpt提上日程

**Comment by zhzLuke96** [comment]
一直没接入的原因：
1. gpt sovits 据我评估社区微调模型比较活跃，基础模型zeroshot能力一般，所以我计划是在 forge 整个系统支持"同一架构下切换不同模型"之后再接入
2. 年底开发资源有限，所以推进的慢 😂

不过，如果确实有需要的话，我估计可以先适配一个不支持切换的简易版本，支持基本的推理应该有时间搞 #198

关于 --compile 这个是可以开的，但是这方面就依赖上游库了，不同的架构可能有不同的问题 😂
比如有的模块可以compile但是compile之后更慢...有的则是直接无法支持，并且一般情况使用 compile 都需要进行推理预热，这个预热时间可能需要几分钟，个人感觉并不是很好的选择
如果有这方面的进展我会关注看看
